### PR TITLE
Fixes #10698: Update server requirement with load test data

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10698

### CODE REVIEW

File: 10_installation/05_requirements/30_hardware_specifications.txt [hunk 1]
```diff
@@ -19,6 +19,15 @@ number of configured components to check, etc...).
 A dedicated server is strongly recommended, either physical or virtual with at least one dedicated core.
 Rudder Server runs on both 32 (if available) and 64 bit versions of every supported Operating System.
 
+[NOTE]
+
+====
+
+Rudder does not fear big infrastructures. It is currently use in production in
```
Comment (amousset) [inline 10_installation/05_requirements/30_hardware_specifications.txt]: used in production on

File: 10_installation/05_requirements/30_hardware_specifications.txt [hunk 2]
```diff
@@ -30,11 +39,28 @@ The required amount of RAM mainly depends on the number of managed nodes. A gene
 When managing more than 1000 nodes, we also recommend you to use a multiserver
 installation for Rudder as described in chapter <<multiserver-rudder, Multiserver Rudder>>.
 
-For large installations, you should also tune the amount of RAM given to:
+When your server has more than 2 Go of RAM, you need to configure the RAM allocated
```
Comment (amousset) [inline 10_installation/05_requirements/30_hardware_specifications.txt]: GB

File: 10_installation/05_requirements/30_hardware_specifications.txt [hunk 3]
```diff
@@ -30,11 +39,28 @@ The required amount of RAM mainly depends on the number of managed nodes. A gene
...
+
+====
+
+As an example, a Rudder server which manages 2600 nodes (with a lot of policies
+checked) will need:
 
-* the web application, as explained in the section
-<<_configure_ram_allocated_to_jetty, about webapplication RAM configuration>>
-* PostgresSQL, as explained in the <<_optimize_postgresql_server, Optimize PostgreSQL Server>> section
+* A server with 8 Go of RAM,
```
Comment (amousset) [inline 10_installation/05_requirements/30_hardware_specifications.txt]: GB

File: 10_installation/05_requirements/30_hardware_specifications.txt [hunk 4]
```diff
@@ -30,11 +39,28 @@ The required amount of RAM mainly depends on the number of managed nodes. A gene
...
+* 4 Go of RAM will be allocated to the JVM.
```
Comment (amousset) [inline 10_installation/05_requirements/30_hardware_specifications.txt]: GB

**Comment by fanf** [comment]
PR rebased

**Pull request merged**